bitnami/opencart/templates/svc.yaml

```diff
@@ -28,5 +30,4 @@ spec:
       {{- if (and (eq .Values.service.type "NodePort") (not (empty .Values.service.nodePorts.https)))}}
```

Comment:
we need to add an else that set `nodePorts: null` when `service.type=ClusterIP|LoadBalancer`, something like https://github.com/bitnami/charts/blob/8ee923ce98e69ef9e4d0f660600d648bda187a0d/bitnami/ghost/templates/svc.yaml#L23 so that we can upgrade the service to change the type

Reply:
Good point, changing